server/protocols/jmap/src/test/java/org/apache/james/jmap/model/mailbox/RoleTest.java

```diff
@@ -20,39 +20,17 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import java.util.Locale;
 import java.util.Optional;
 
 import org.junit.Test;
 
 public class RoleTest {
 
-    @Test
-    public void fromShouldReturnInbox() {
-        assertThat(Role.from("inbox")).isEqualTo(Optional.of(Role.INBOX));
-    }
-
     @Test
     public void fromShouldReturnEmptyWhenUnknownValue() {
         assertThat(Role.from("jjjj")).isEqualTo(Optional.empty());
     }
 
-    @Test
-    public void fromShouldReturnInboxWhenContainsUppercaseValue() {
-        assertThat(Role.from("InBox")).isEqualTo(Optional.of(Role.INBOX));
-    }
-
-    @Test
-    public void fromShouldReturnInboxWhenContainsUppercaseValueInTurkish() {
```

Comment:
you lost this very good test case

Reply:
We do not need this anymore, we are never converting the string case.

Reply:
Why can't we restore this test in the final diff?

Reply:
Ok, I will add it; but it has no real added value as we are no more changing the case of the string.